packages/babel-plugin-proposal-optional-chaining/src/index.js

```diff
@@ -23,14 +23,23 @@ export default declare((api, options) => {
 
     visitor: {
       "OptionalCallExpression|OptionalMemberExpression"(path) {
-        const { parentPath, scope } = path;
+        const { scope } = path;
+        let parentPath;
+        // unwrap parenthesis around parent path
+        for (
+          { parentPath } = path;
+          parentPath.type === "ParenthesizedExpression";
+          { parentPath } = parentPath
+        );
```

Comment:
Maybe we should call it something other than `parentPath`?

```suggestion
        const parentPath = path.findParent(p => !p.isParenthesizedExpression());
```